CodeStarterCamp_Week3/main.swift

```diff
@@ -9,33 +9,49 @@
 import Foundation
 
 class Person {
-    var money: Int = 0
+    var money: Int
     
-    func buyCoffee() {
-        print("buy coffee.")
+    init(money: Int) {
+        self.money = money
+    }
+    
+    func order(_ coffee: Coffee, of coffeeshop: CoffeeShop, by name: String) {
+        coffeeshop.make(coffee, for: name)
+        if let coffeePrice: Int = coffeeshop.menuTable[coffee] {
+            if money >= coffeePrice {
+                money -= coffeePrice
+            } else {
+                print("잔액이 \(coffeePrice-money)만큼 부족합니다.")
```

Comment:
다른 부분과 컨벤션이 다른 거 같아요!
```suggestion
                print("잔액이 \(coffeePrice - money)만큼 부족합니다.")
```